definitions/fenix.toml

```diff
@@ -371,3 +378,9 @@ from_expression = """(
 experiments_column_type = "simple"
 client_id_column = "client_id"
 submission_date_column = "submission_date"
+
+[data_sources.new_profile_activation]
+from_expression = "`moz-fx-data-shared-prod.fenix.new_profile_activation`"
+experiments_column_type = "simple"
```

Comment:
I think we might want `experiments_column_type = "none"` here. The [other options](https://github.com/mozilla/mozanalysis/blob/main/src/mozanalysis/metrics/__init__.py#L105-L128) are for tables enriched with experiment information and used primarily to filter out metrics from the day of enrollment. I don't think we need to worry about that here